### PR TITLE
Add missing aria label for campaign close button

### LINKED ISSUE #11859: Aria label missing for close button
**Describe the bug**

Close button in the banner has icon but does not have corresponding text associated with it so screen reader user might not know what the button does clearly. Adding aria label will help

**To Reproduce**

1. Go to https://dev.to/
2. Make sure it campaign banner at the top
3. Now try navigating the page with screen reader to see what the close button in the banner say

**Expected behavior**

When focused with screen reader on the close button should describe what it does

**Screenshots**

closeButton

**Desktop:**

- OS, version: Ubuntu 20.04
- Browser, version: Version 1.15.75 Chromium: 86.0.4240.99 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by andreamartz**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fixes an accessibility issue with campaign close button

## Related Tickets & Documents
Closes #11859

## QA Instructions, Screenshots, Recordings

1. Login as an administrator with edit rights to the configuration of Forem.
2. Create a campaign by following the instructions in the [admin guide](https://forem.gitbook.io/forem-admin-guide/misc/customizing-with-campaigns-+-display-ads).
3. Test with a screen reader by focusing on the close button. A tool like Voice Over on Mac should work as a screen reader. The screen reader should say, "Close campaign banner."

### UI accessibility concerns?

This PR fixes an accessibility issue with the close button.

## Added tests?

- [ ] Yes
- [x] No, and this is why: Tested this manually with a screen reader.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/javascripts/initializers/initializeHeroBannerClose.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 function initializeHeroBannerClose() {
   let bannerWrapper = document.getElementById('hero-html-wrapper');
   let closeIcon = document.getElementById('js-hero-banner__x');
+  closeIcon.setAttribute('aria-label', 'Close campaign banner');
```
Comment (nickytonline) [inline app/assets/javascripts/initializers/initializeHeroBannerClose.js:6]: I looked through the code, and this seems to be the only reference to `js-hero-banner__x`, so this seems fine adding the aria label here.

**Review by nickytonline — APPROVED** [review]
Tested this locally with Voice Over and all is good. Thanks for the PR @andreamartz! 🚀

**Review by katiedavis — APPROVED** [review]
🏅Thank you!!

File: app/assets/javascripts/initializers/initializeHeroBannerClose.js (hunk 1 above)
Comment (nickytonline) [inline app/assets/javascripts/initializers/initializeHeroBannerClose.js:6]: Refactor (blocking): Move `setAttribute` into conditional check for `closeIcon`. I tested the happy path and clearly haven't had enough coffee. We need to check if the element exists before setting the attribute.

```js
  if (bannerWrapper && closeIcon) {
    closeIcon.setAttribute('aria-label', 'Close campaign banner');
...
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
@andreamartz I tested the happy path and clearly had a Friday moment. We need to check if the element exists before setting the attribute, so just a small change request.

**Review by nickytonline — APPROVED** [review]
Thanks for making the change @andreamartz! 🚀

**Pull request merged**